Makefile

```diff
@@ -397,30 +396,29 @@ ci-clean:
 	podman $(PODMAN_REMOTE_ARGS) \
 	    image prune --all --filter="label=aro-*=true"
 
+.PHONY: version
+version:
+	echo "RP $(LOCAL_ARO_RP_IMAGE):$(VERSION)"
+
```

Comment:
do we need this, or can it be removed?

Reply:
I have removed the echo "RP $(LOCAL_ARO_RP_IMAGE):$(VERSION)"